tests/unit/test_launchers.py

```diff
@@ -105,10 +106,22 @@ def test_launcher_deleted_remove_tempdir(self):
 
 
 class TestFennecLauncher(unittest.TestCase):
+
+    def setUp(self):
+        self.test_root = '/sdcard/tmp'
+        self.profile = Profile()
+        self.profile_name = os.path.basename(self.profile.profile)
```

Comment:
It seems that this variable is only used after - could you not attach it to self - or even better do not use a variable ?